skyvern/core/script_generations/generate_script.py

```diff
@@ -420,12 +420,12 @@ def _action_to_stmt(act: dict[str, Any], task: dict[str, Any], assign_to_output:
                     comma=cst.Comma(),
                 )
             )
-    intention = act.get("intention") or act.get("reasoning") or ""
+    intention = act.get("intention") or act.get("prompt") or act.get("reasoning") or ""
```

Comment:
our action model doesn't have the "prompt" field so no need to change this.

```suggestion
    intention = act.get("intention") or act.get("reasoning") or ""
```